examples/link.rs

```diff
@@ -0,0 +1,28 @@
+use dioxus::prelude::*;
+
+fn main() {
+    dioxus::desktop::launch(app);
+}
+
+fn app(cx: Scope) -> Element {
+    cx.render(rsx! (
+        div {
+            p {
+                a {
+                    href: "http://dioxuslabs.com/",
+                    "default link"
+                }
+            }
+            p {
+                a {
+                    href: "http://dioxuslabs.com/",
+                    "browser-open": "false",
+                    onclick: |_| {
+                        println!("Hello Dioxus");
+                    },
+                    "custom event link",
+                }
```

Comment:
instead of "browser-open", let's just hijack "prevent_default" . That way the "default" behavior in desktop is to open links remotely.



so the second example should be 

```rust
                a {
                      href: "http://dioxuslabs.com/",
                      prevent_default: "true",
                      onclick: |_| {
                          println!("Hello Dioxus");
                      },
                      "custom event link",
                  }
```

Reply:
got it!